basics/data-import/segment-upload.md

```diff
@@ -0,0 +1,99 @@
+---
+description: Upload a table segment in Apache Pinot.
+---
+
+# Upload a table segment from an external source
+
+This procedure uploads one or more table segments that have been stored as Pinot segment binary files outside of Apache Pinot, such as if you had to close an original Pinot cluster and create a new one.
+
+Choose one of the following:
+
+* If your data is in a location that uses HDFS, create a segment fetcher.
+* If your data is on a host where you have SSH access, use the Pinot Admin script.
```

Comment:
is "admin" really a proper noun here?

Reply:
The script seems to be called "Pinot Admin".